src/index.html

```diff
@@ -8,106 +8,109 @@
     <meta name="description" content="Essential JS 2" />
     <meta name="author" content="Syncfusion" />
     <link rel="shortcut icon" href="resources/favicon.ico" />
-    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i"
-        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" 
+    rel="stylesheet">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
     <!--style reference from node_modules/@syncfusion/ej2/-->
-    <link href="/styles/styles.css" rel="stylesheet" />
-
-    <script src="node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
+    <link href="./styles/styles.css" rel="stylesheet" />
+    <script src="/node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
     <script src="system.config.js" type="text/javascript"></script>
-
+    <link rel="stylesheet" href="/node_modules/@syncfusion/ej2-navigations/styles/material.css">
```

Comment:
Move this import state to styles.css,
avoid unnecessary alignment changes

Reply:
Moved imports into separate stylesheet.